web/js/modules/animation/actions.js

```diff
@@ -10,12 +10,29 @@ import {
   UPDATE_END_DATE,
   TOGGLE_GIF
 } from './constants';
+import util from '../../util/util';
 
 export function onActivate() {
   return (dispatch, getState) => {
-    const { compare, date } = getState();
+    const { compare, date, animation } = getState();
     const dateStr = compare.isCompareA ? 'selected' : 'selectedB';
-    dispatch({ type: OPEN_ANIMATION, date: date[dateStr] });
+    const activeDate = date[dateStr];
+    if (!animation.startDate || !animation.endDate) {
+      const sevenDaysBefore = util.dateAdd(activeDate, 'day', -7);
+      const severDaysAfter = util.dateAdd(activeDate, 'day', 7);
```

Comment:
change `severDaysAfter` to `severDaysAfter` in a few places

Reply:
good catch